test/images/agnhost/grpc-health-checking/grpc-health-checking.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package grpchealthchecking offers a tiny grpc health checking endpoint.
+package grpchealthchecking
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/health/grpc_health_v1"
+	"google.golang.org/grpc/status"
+)
+
+// CmdGrpcHealthChecking is used by agnhost Cobra.
+var CmdGrpcHealthChecking = &cobra.Command{
+	Use:   "grpc-health-checking",
+	Short: "Starts a simple grpc health checking endpoint",
+	Long:  "Starts a simple grpc health checking endpoint --port to serve on. The endpoint returns SERVING for the first --delay-unhealthy-sec, and NOT_SERVING after this.",
```

Comment:
the KEP also mentions 
>   // The service name can be the empty string (i.e. "").                         //+
>  Service string `json:"service" protobuf:"bytes,2,opt,name=service"`  

do we want to add an option to test by service too?

Reply:
right, good suggestion. Adding...